### PR TITLE
Additional Info and HR-Monitor GUI upgrade

### PR DESCRIPTION
**Pull request by dalex135**
@vishwasanavatti coded the frontend part and I coded the backend part. Me and @Nasser-Eddin-Nasser merged the createAdmin branch with this branch.

### CODE REVIEW

File: Backend/hs_err_pid8816.log [hunk 1]
```diff
@@ -0,0 +1,745 @@
...
+CPU:total 4 (initial active 4) (2 cores per cpu, 2 threads per core) family 6 model 58 stepping 9, cmov, cx8, fxsr, mmx, sse, sse2, sse3, ssse3, sse4.1, sse4.2, popcnt, avx, aes, clmul, erms, ht, tsc, tscinvbit, tscinv
+
+Memory: 4k page, system-wide physical 8059M (668M free)
+TotalPageFile size 8059M (AvailPageFile size 0M)
+current process WorkingSet (physical memory assigned to process): 397M, peak: 434M
+current process commit charge ("private bytes"): 460M, peak: 504M
+
+vm_info: Java HotSpot(TM) 64-Bit Server VM (12.0.2+10) for windows-amd64 JRE (12.0.2+10), built on Jul 16 2019 01:56:17 by "mach5one" with MS VC++ 15.5 (VS2017)
+
+END.
```
Comment (vishwasanavatti) [inline Backend/hs_err_pid8816.log]: is this file necessary?

File: Backend/replay_pid8816.log [hunk 2]
```diff
@@ -0,0 +1,9553 @@
...
+instanceKlass org/jetbrains/plugins/gradle/tooling/ModelBuilderService
+instanceKlass org/jetbrains/plugins/gradle/tooling/ModelBuilderContext
+instanceKlass org/jetbrains/plugins/gradle/tooling/internal/ExtraModelBuilder
+instanceKlass RegistryProcessor
+instanceKlass java/util/zip/ZipFile$ZipEntryIterator
+instanceKlass java/util/jar/JarFile$$Lambda$232
+instanceKlass org/gradle/internal/instantiation/generator/Jsr330ConstructorSelector$CachedConstructor
+instanceKlass org/gradle/internal/instantiation/generator/InjectUtil
+instanceKlass org/gradle/internal/instantiation/generator/AsmBackedClassGenerator$NoOpBuilder
+inst
```
Comment (vishwasanavatti) [inline Backend/replay_pid8816.log]: I guess these log files are pushed by mistake. Please remove

File: Backend/src/main/java/Digitact/Backend/ConfigProperties.java [hunk 3]
```diff
@@ -4,7 +4,8 @@
 
 @Configuration
 public class ConfigProperties {
-    public static final String absoluteFileSystemPath = "xx\\xx\\";
+    public static final String absoluteFileSystemPath =
+            "\\\\D:\\Masters\\2nd Semester\\AMOS Project\\FileSystem\\";
```
Comment (vishwasanavatti) [inline Backend/src/main/java/Digitact/Backend/ConfigProperties.java]: Please remove the hard code path.

File: Backend/src/main/java/Digitact/Backend/Model/User/ApplicantUI.java [hunk 4]
```diff
@@ -106,6 +110,14 @@ public Title getTitle() {
         return title;
     }
 
+    public String getAdditionalInfo() {
+        return additionalInfo;
+    }
+
+    public void setAdditionalInfo(String impression) {
+        this.additionalInfo = impression;
+    }
```
Comment (vishwasanavatti) [inline Backend/src/main/java/Digitact/Backend/Model/User/ApplicantUI.java]: It is not impression. it is just additional information that is being given applicant. Can you give different name?

**Review by vishwasanavatti — CHANGES_REQUESTED** [review]
Overall it looks great. I have added some comments, have a look. Also, Have you set additional information to json structure to send it to server, I dont see that change (you need to add one line in applications-upload.ts file). 
If Nasser PR is merged with this then It would be good to remove the other PR.

File: Backend/hs_err_pid8816.log (hunk 1 above)
Comment (dalex135) [reply Backend/hs_err_pid8816.log]: Added by mistake. Really good that noted it.

**Review by Nasser-Eddin-Nasser — CHANGES_REQUESTED** [review]
HTTP 403 error between FE and BE-server

**Review by Nasser-Eddin-Nasser — APPROVED** [review]
I ran the code locally, test all the functions, and fixed a small bug. 
It can be merged to the master!

**Review by vishwasanavatti — APPROVED** [review]
